lib/analyzer/static-analyzer.ts

```diff
@@ -47,20 +43,13 @@ import { ImageAnalysis, OSRelease, StaticAnalysis } from "./types";
 
 const debug = Debug("snyk");
 
-const supportedArchives: ImageType[] = [
-  ImageType.DockerArchive,
-  ImageType.OciArchive,
-];
-
 export async function analyze(
   targetImage: string,
   dockerfileAnalysis: DockerFileAnalysis | undefined,
-  options: StaticAnalysisOptions,
+  imageType: ImageType,
+  imagePath: string,
+  globsToFind: { include: string[]; exclude: string[] },
 ): Promise<StaticAnalysis> {
-  if (!supportedArchives.includes(options.imageType)) {
```

Comment:
Have we agreed that this is not required any more? As far as I can remember that error was requested to throw if image type is unknown, just an FYI

Reply:
This was only exercised in the static scanning if whoever called it with `staticScanningOptions` gave a bad image type. In reality this never happened in the CLI for example. 

We already have code up the stack that throws if ImageType is not the one we expect 🙏